e2e/tests/121-Portal/MakeNewPayment/OCW/SendPaymentInstructionsOCW.spec.ts

```diff
@@ -46,7 +46,7 @@ test.beforeEach(async ({ page }) => {
   );
 });
 
-test('[28448] OCW: Send payment instructions', async ({ page }) => {
+test.skip('[28448] OCW: Send payment instructions', async ({ page }) => {
```

Comment:
It would be best to add a reason why we skip a test whenever we skip. Ideally also a link to a task to work on re-enabling the test.